WooCommerce/Classes/Extensions/NumberFormatter+Localized.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+extension NumberFormatter {
```

Comment:
Non-blocking question: since we already have a `NumberFormatter` extension, do you think it is worthwhile to aggregate everything under a single extension, just not to create too many extensions of the same class?

Reply:
Personally I don't see having separate extensions of the same type is a problem - since each extension deals with a different problem (which is named in the files). So I'm keeping it as is and merging this PR.